### ISSUE TITLE
song "name" not displayed

### ISSUE THREAD

**Issue by MStrecke** [CDrummond/cantata#965]
**Versions:**
- mpd 0.20.4
- Cantata 2.0.1
- Ubuntu 14.04

I'm using mpd to listen to internet radio.
The station links are stored in the mpd playlist folder as one extended M3U file with the station names as comment in the EXTINF tag.
mpd seems to read this info as it is sent to Cantata as answer to a _listplaylistinfo_ command.

A quick look into the source suggests that this info is being parsed, but the "Name" is not shown in the GUI.

Is this intentional or did I miss a setting somewhere?

Michael

allradio.m3u

```
#EXTM3U
#EXTINF:-1,FIP
http://direct.fipradio.fr/live/fip-midfi.mp3?ID=f9fbk29m84
#EXTINF:-1,RTBF VivaCité Liège
http://icepe3.infomaniak.ch:80/vivaliege-128.mp3
#EXTINF:-1,RTBF La Première
http://icepe4.infomaniak.ch:80/lapremiere-128.mp3
#EXTINF:-1,RTBF Musiq3
http://icepe9.infomaniak.ch:80/musiq3-128.aac
...
```

Wireshark capture

```
...
listplaylistinfo "allradio"

file: http://direct.fipradio.fr/live/fip-midfi.mp3?ID=f9fbk29m84
Name: FIP
Time: 0
duration: 0.000
file: http://icepe3.infomaniak.ch:80/vivaliege-128.mp3
Name: RTBF VivaCit.. Li..ge
Time: 0
duration: 0.000
file: http://icepe4.infomaniak.ch:80/lapremiere-128.mp3
Name: RTBF La Premi..re
Time: 0
duration: 0.000
file: http://icepe9.infomaniak.ch:80/musiq3-128.aac
Name: RTBF Musiq3
Time: 0
...
```

![cantata](https://cloud.githubusercontent.com/assets/5202211/23187969/37973628-f88d-11e6-8e15-adb29ed18022.png)

**Comment by CDrummond** [comment]
Not sure why 'Name' is not being handled here, but I'll look into fixing this.

However, Cantata (and MPDroid) using a special playlist named '[Radio Streams]' (without the quotes) to store radio streams. Each entry has the format: <url>#<name> - e.g. 

htttp://somewhere.com#SomeWhere Radio

The reason for this is that (AFAIK) when using MPD's protocol to add an entry to a playlist you can only add the url - hence the name is added as an anchor to the url.

If you use this playlist, it will appear as 'Favourites' under Internet/Streams

**Comment by MStrecke** [comment]
Thank you for the work-around.

**Comment by CDrummond** [comment]
Well, its not really a workaround - its the way Cantata is designed to store streams. 

However, it does not fix this issue - which as I stated, I will look into fixing.

**Comment by CDrummond** [comment]
Fixed in master for 2.1.0
